cockroachcloud/user-authorization.md

```diff
@@ -1,10 +1,21 @@
 ---
-title: User Authorization
-summary: Learn about the user authorization features for CockroachDB Cloud clusters.
+title: Managing User Authorization in CockroachDB Cloud
+summary: Learn procedures for managing the life cycle of CockroachDB Cloud users and roles.
 toc: true
 docs_area: manage
 ---
 
+{{ site.data.products.db }} has a two-level authorization model: users have permissions within organizations, and on specific database clusters within organizations.
+
+Before proceding, it is recommended to review the {{ site.data.products.db }} authorization model, which is detailed in full here: [Authorization in {{ site.data.products.db }}](authorization.html)
+
+This page documents procedures for managing the life cycle of CockroachDB Cloud users and roles, including:
+
+- Adding users to your organization, by creating them or inviting them from another organization
+- managing organization-level users roles
+- granting organization users access to clusters through cluster roles
+- managing the privileges associated with cluster roles
+
 {{ site.data.products.db }} requires you to create SQL users to access the cluster.
 
 By default, a new SQL user created using a [Console Admin](console-access-management.html#console-admin) is assigned to the `admin` role. An `admin` SQL user has full privileges for all databases and tables in your cluster. This user can also create additional users and grant them appropriate privileges.
```

Comment:
Should this info also be in `authorization.md`?

Reply:
nah, i don't think so, that's more about the model imo. in any case, this is changing with the upcoming release so it'll be moot.